application.yml

```diff
@@ -38,6 +38,10 @@ spring:
         show_sql: true
         use_sql_comments: true
         format_sql: true
+  servlet:
+    multipart:
+      location: "${java.io.tmpdir}"
+      file-size-threshold: 128KB
```

Comment:
You are correct this is the threshold to hold the file in memory or dump it to the location if it crosses that threshold.  This is easy to config through the iat config repo.  It goes in the ap-ims.yml file.